pkg/destroy/gcp/cloudcontroller.go

```diff
@@ -22,7 +22,7 @@ func (o *ClusterUninstaller) listCloudControllerInstanceGroups() ([]cloudResourc
 func (o *ClusterUninstaller) listCloudControllerBackendServices(instanceGroups []cloudResource) ([]cloudResource, error) {
 	urls := sets.NewString()
 	for _, instanceGroup := range instanceGroups {
-		urls.Insert(o.getInstanceGroupURL(instanceGroup))
+		urls.Insert(instanceGroup.url)
```

Comment:
This is good... please delete the getInstanceGroupURL function definition as well. It doesn't appear to be used after this change.
https://github.com/openshift/installer/blob/a7f8c8da45993927639ca16c725e04d9000bdb0f/pkg/destroy/gcp/gcp.go#L186-L189